ui/admin/tests/acceptance/host-catalogs/hosts/list-test.js

```diff
@@ -94,6 +98,6 @@ module('Acceptance | host-catalogs | hosts | list', function (hooks) {
     instances.hostCatalog.authorized_collection_actions.hosts = ['create'];
     await visit(urls.hostCatalog);
     assert.ok(find(`[href="${urls.hosts}"]`));
-    assert.ok(find(`.rose-layout-page-actions [href="${urls.newHost}"]`));
+    assert.dom(MANAGE_DROPDOWN_SELECTOR).exists();
```

Comment:
I think this test is verifying that we will still render the "New Host" action in the dropdown with only the `create` permission for hosts. We likely need to click the manage dropdown to open it, then do something like `assert.ok(find('[href="${urls.newHost}"]')`. The newer way of checking for the href that won't have a false positive is `assert.dom(YOUR_SELECTOR).hasAttribute('href', urls.newHost)`

Reply:
Thanks for the check Cameron!